python/core/auto_generated/geometry/qgsgeometry.sip.in

```diff
@@ -1810,6 +1816,9 @@ by calling :py:func:`~QgsGeometry.lastError` on the returned geometry.
 .. note::
 
    this operation is not called union since its a reserved word in C++.
+
+:param geometry: geometry to perform the operation
+:param gridSize: If this optional argument is provided, the inputs are snapped to a grid of the given size, and the result vertices are computed on that same grid. (Requires GEOS-3.9.0 or higher) /since 3.28
```

Comment:
```suggestion
:param gridSize: If this optional argument is provided, the inputs are snapped to a grid of the given size, and the result vertices are computed on that same grid (since 3.28).
```

Reply:
(elsewhere too)